worlds/sc2wol/PoolFilter.py

```diff
@@ -18,73 +21,103 @@
 FACTORY_UNITS = {"Hellion", "Vulture", "Goliath", "Diamondback", "Siege Tank", "Thor", "Predator", "Widow Mine"}
```

Comment:
Just noticed that the Cyclone is missing from the list of factory units.

Reply:
Fixed